tests/app/templating/summary/test_question.py

```diff
@@ -136,6 +136,8 @@ def test_checkbox_button_other_option_empty(self):
         self.assertEqual(len(question.answers[0].value), 1)
         self.assertEqual(question.answers[0].value[0], 'Other option label')
 
+
```

Comment:
Is it worth removing these two lines otherwise there will be an unnecessary touch on the file in the git history?